src/llmcompressor/transformers/sparsification/sparse_model.py

```diff
@@ -62,6 +62,13 @@ def from_pretrained(
             pretrained_model_name_or_path directory and applied if found
         :return the created model for causal language modeling
         """
+        logger.info(
```

Comment:
In this file, seems like you're still using SparseAutoModel in `text_generation_from_pretrained` pathway?